client/lib/mixins/url-search/component.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import assign from 'lodash/assign';
```

Comment:
We're using a Babel transform to improve loading from `lodash` and so should use `import { assign } from 'lodash'` instead.

Why are we importing `assign` anyway? We're using `Object.assign()` in Calypso and that is safe to use